roles/openshift_logging/library/openshift_logging_facts.py

```diff
@@ -77,9 +77,9 @@ def oc_command(self, sub, kind, namespace=None, name=None, add_options=None):
             process = Popen(cmd, stdout=PIPE, stderr=PIPE)   # noqa: F405
             out, err = process.communicate(cmd)
             if len(err) > 0:
-                if 'not found' in err:
+                if 'not found' in str(err):
                     return {'items': []}
-                if 'No resources found' in err:
+                if 'No resources found' in str(err):
                     return {'items': []}
                 raise Exception(err)
```

Comment:
Does `Exception` take a `byte` object as well as a `str`?
Is it possible that the `str(err)` conversion could throw an Exception e.g. attempting to convert a string to utf8 but the byte array is not a valid utf8 string.

Reply:
1. Yes `Exception` takes a string as well. So that will be fine.
2. Yes, it can fail, good catch. I'll change to `decode` with an error handler instead.